src/test/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileConcurrencyITest.java

```diff
@@ -136,6 +136,20 @@ void shouldDeleteLegacyUnversionedDocument() {
         assertTrue(actual.isEmpty());
     }
 
+    @Test
+    void shouldUpdateVersionedDocumentWhenPatchLinksRequest() {
+        // given
+        VersionedCompanyProfileDocument document = buildCompanyProfileDocument();
+
+        document = companyProfileRepository.insert(document);
+        assertEquals(0, document.getVersion());
+
+        // when
+//        companyProfileService.processLinkRequest();
```

Comment:
Does this test need to be removed or completed?

Reply:
was rushing sorry removed now - covered in the E2EItests